device/bgp/bgpcapability.yaml

```diff
@@ -170,3 +170,41 @@ components:
           type: boolean
           default: false
           x-field-uid: 25
+        ipv4_mpls:
+          description: >-
+            For Single Label binding to the IPv4 address prefix, BGP speaker will advertise (AFI, SAFI) pair capability as (1, 4) in BGP Open Message under the Multiprotocol 
+            extensions capability.
+          type: boolean
+          default: false
+          x-field-uid: 26
+        ipv6_mpls:
+          description: >-
+            For Single Label binding to the IPv4 address prefix, BGP speaker will advertise (AFI, SAFI) pair capability as (2, 4) in BGP Open Message under the Multiprotocol 
+            extensions capability.
+          type: boolean
+          default: false
+          x-field-uid: 27
+        ipv4_mpls_multi:
+          description: >-
+            The MPLS multiple Lable capability is advertised in Optional Capability under Multiple Label Capability (code 8).
+            Reference: https://datatracker.ietf.org/doc/html/rfc8277#section-2.1.
+            For Multiple Labels binding to the IPv4 address prefix, BGP speaker will advertise triple of the form <AFI=1, SAFI=4, Count=n> in BGP Open Message under the Multiprotocol 
+            extensions capability, where n = ipv4_mpls_multi, the numbers of labels.
```

Comment:
How to disable this ( which should be the default ) . Maybe need an object with or without enable ? [ Same for ipv6_multi]